build.gradle

```diff
@@ -23,9 +23,7 @@ allprojects {
         flatDir {
             dirs project(projectName).file('libs')
```

Comment:
i think we can remove also this block of `flatDir`

Reply:
fixed